src/View/Helper/FlashMessengerFactory.php

```diff
@@ -8,6 +8,7 @@
 use Laminas\ServiceManager\Factory\FactoryInterface;
 use Laminas\ServiceManager\ServiceLocatorInterface;
 
+use function is_array;
 use function method_exists;
 
 class FlashMessengerFactory implements FactoryInterface
```

Comment:
Changes to this factory require test additions

Reply:
Hmm, tests exist, but mixed up with all the rest 🤔

Reply:
> Hmm, tests exist, but mixed up with all the rest 🤔

I separated 1 test to 2